src/jit/compiler.cpp

```diff
@@ -9246,12 +9244,9 @@ int cTreeFlagsIR(Compiler* comp, GenTree* tree)
                     chars += printf("[VAR_CSE_REF]");
                 }
 #endif
-                if (op == GT_REG_VAR)
+                if (tree->gtFlags & GTF_REG_BIRTH)
                 {
-                    if (tree->gtFlags & GTF_REG_BIRTH)
-                    {
-                        chars += printf("[REG_BIRTH]");
-                    }
+                    chars += printf("[REG_BIRTH]");
```

Comment:
This flag isn't really in use, but #18196 will take care of eliminating it.

Reply:
This entire if needs to go away. The flag only applies to GT_REG_VAR nodes so attempting to print it for other nodes will result in misleading dumps.

Reply:
Will remove.